lib/travis/api/build/app.rb

```diff
@@ -20,16 +20,17 @@ class App < Sinatra::Base
         end
 
         configure do
-          use Sentry unless Travis::Build.config.sentry_dsn.empty?
-          use Metriks unless Travis::Build.config.librato.email.empty? ||
-                             Travis::Build.config.librato.token.empty? ||
-                             Travis::Build.config.librato.source.empty?
+          use Sentry unless Travis::Build.config.sentry_dsn.nil?
+          use Metriks unless Travis::Build.config.librato.email.nil? ||
+                             Travis::Build.config.librato.token.nil? ||
+                             Travis::Build.config.librato.source.nil?
```

Comment:
If any of these things is `nil`, could it be that the corresponding key is missing in the config?

If so, I suspect this is a general problem for any key missing in the config. A couple of solutions come to mind.

1. Coerce config values to Strings before calling `#empty?`; e.g., `Travis::Build.config.sentry_dsn.to_s.empty?`
1. Modify `Travis::Build` so that `""` is returned when the key is missing in the config. I think this is a divergence from the general `Hashr` behavior, so this is less preferred.